### ISSUE TITLE
Autokeras

### ISSUE THREAD

**Issue by JunGyuRyu** [simple-icons/simple-icons#10324]
### Brand Name

autokeras

### Website

https://autokeras.com/

### Popularity Metric

Similar rank: 6,210

https://www.similarweb.com/website/autokeras.com/#overview

### Official Resources for Icon and Color

sry, idk..

**Comment by PeterShaggyNoble** [comment]
Thanks for the request, @JunGyuRyu, but this is well outside our scope based on its SImilarweb rank. If there's another metric we can use to determine its popularity - perhaps something from #10114 - then we'll be happy to revisit this issue.

**Comment by JunGyuRyu** [comment]
Thank you for kind reply!
